toolkit/tools/pkg/isomakerlib/isomaker.go

```diff
@@ -1,7 +1,7 @@
 // Copyright Microsoft Corporation.
 // Licensed under the MIT License.
 
-package main
+package isomakerlib
```

Comment:
Just `isomaker` to follow how we've been naming our packages elsewhere. Similarly, we don't add `pkg` suffixes.
```suggestion
package isomaker
```

Reply:
Some people on the Mariner team have been advocating for putting reusable code under the `pkg` directory. (There was a PR a while back to do a mass move into `pkg`. But it stalled out.)

The `lib` suffix is added to ensure the package directory name is unique. It can be quite annoying having to rename module in import statements to avoid name conflicts.

(This is also following existing precedent set by the `imagecustomizer` tool.)

Reply:
@PawelWMS plz, let me know if you feel strongly on this one.